iree-prof-tools/iree-prof-output.cc

```diff
@@ -24,13 +24,14 @@ ABSL_FLAG(std::string, output_chrome_file, "",
           "or conversion.");
 ABSL_FLAG(bool, output_stdout, true,
           "Whether to print Tracy result to stdout.");
-ABSL_FLAG(std::vector<std::string>, zone_substrs,
-          std::vector<std::string>({"iree_hal_buffer_map_", "_dispatch_"}),
-          "Comma-separated substrings of tracy zones to output to stdout. "
-          "If empty, no zones will be output.");
-ABSL_FLAG(std::vector<std::string>, thread_substrs, {},
-          "Comma-separated substrings of threads to output to stdout. "
-          "If empty, all thread including main threads will be output.");
+ABSL_FLAG(bool, output_zones_stdout, true,
+          "Whether to print Tracy result of individual zones to stdout.");
+ABSL_FLAG(bool, output_ops_stdout, true,
+          "Whether to print Tracy result of ML operation to stdout.");
+ABSL_FLAG(std::string, zone_regex, "iree_hal_buffer_map_|_dispatch_",
```

Comment:
I think we can use iree_hal_buffer_map_[zero|fill|read|write|copy] to exclude iree_hal_buffer_map_range, which is called by the former functions. (My regex syntax might be wrong though.)

Also we can use `_dispatch_[0-9]+_` to filter the dispatch functions only. There are other functions caught with the current pattern.